packages/react-core/src/components/Modal/Modal.tsx

```diff
@@ -15,9 +15,9 @@ export interface ModalProps extends React.HTMLProps<HTMLDivElement> {
   /** Complex header (more than just text), supersedes title for header content */
   header?: React.ReactNode;
   /** Simple text content of the Modal Header, also used for aria-label on the body */
-  title: string;
-  /** Flag to hide the title */
-  hideTitle?: boolean;
```

Comment:
This looks great! Can you please add all the breaking changes to the bottom of your PR? in this form:
```
## Breaking changes
- Your list of changes.
- You can use MD
```